Tree_From_Preorder_and_Inorder_Traversal_in_Python.py

```diff
@@ -0,0 +1,70 @@
+
+#constructing a node class for tree
+
+class Node:
+    def __init__(self):
+        self.data=None
+        self.left=None
+        self.right=None
+
+#constructing a binary tree class
+
+class BinaryTree:
+    def __init__(self,preorder,ion) :
+        self.root = constructPre(preorder, 0, len(preorder) - 1, ino, 0, len(ino) - 1)
+
+#function for constructing the tree from given inorder and preorder traversal
+
+def constructPre(pre,plo,phi,ino,ilo,ihi):
```

Comment:
Please use more understandable variable names and make two different program for inorder and preorder.